backend/src/common/models/user.ts

```diff
@@ -1,3 +1,23 @@
 export interface IAuthUser {
   id: string;
 }
+
+export interface IRegisterUser {
+  fullName: string;
+  email: string;
+  password: string;
+}
+
+export interface IUser {
+  id: string,
+  fullName: string,
+  email: string,
+  password: string,
+  displayName: string,
+  imageUrl: string,
+  title: string
+}
+
+export interface IUserResponse extends IUser {
+  token: string,
+}
```

Comment:
create folder common/models/user and create a file for every interface there. Name files by their interface names